### PR TITLE
Fix Multilingual uniform title test

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Broken tests are bad.

## How was this change tested?
Unit, sdr-deploy

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000):
  Success:   98153 (98.153%)
  Different: 1298 (1.298%)
  To Cocina error:     18 (0.018%)
  To Fedora error:     0 (0.0%)
  Missing:     531 (0.531%)
```

After:
```
Status (n=100000):
  Success:   97618 (97.618%)
  Different: 1049 (1.049%)
  To Cocina error:     18 (0.018%)
  To Fedora error:     784 (0.784%)
  Missing:     531 (0.531%)
```

**Comment by justinlittman** [comment]
The churn in this PR is from always adding an (optional) `type: name`. The logic for making it optional was problematic and complicated.

Would like to be reviewed by an Infra dev and @arcadiafalcone.

File: spec/services/cocina/from_fedora/descriptive/contributor_spec.rb [hunk 1]
```diff
@@ -39,7 +39,8 @@
           {
             "name": [
               {
-                "value": 'Dunnett, Dorothy'
+                "value": 'Dunnett, Dorothy',
+                type: 'name'
```
Comment (arcadiafalcone) [inline spec/services/cocina/from_fedora/descriptive/contributor_spec.rb:43]: The `name` type should be applied only if the name is part of a structuredValue.

**Review by jcoyne — APPROVED** [review]
The code looks good to me. Feel free to merge if @arcadiafalcone signs off on the structural changes.

**Pull request closed without merging**